newrelic/hooks/mlmodel_openai.py

```diff
@@ -12,29 +11,31 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-import openai
 import uuid
+
+import openai
+
 from newrelic.api.function_trace import FunctionTrace
-from newrelic.common.object_wrapper import wrap_function_wrapper
-from newrelic.api.transaction import current_transaction
 from newrelic.api.time_trace import get_trace_linking_metadata
-from newrelic.core.config import global_settings
+from newrelic.api.transaction import current_transaction
 from newrelic.common.object_names import callable_name
+from newrelic.common.object_wrapper import wrap_function_wrapper
 from newrelic.core.attribute import MAX_LOG_MESSAGE_LENGTH
```

Comment:
This should have been removed in the sync PR, but we probably don't need to do this truncation check now within the instrumentation. Truncation of attributes to 4096 will be handled in a subsequent PR.